Tests/Marketplace/core_packs_mpv2_list.json

```diff
@@ -23,10 +21,7 @@
     "Phishing",
```

Comment:
@maimorag We Need to remove from MPv2 Phishing as well.
See the ticket comments